stardis/radiation_field/opacities/opacities_solvers/base.py

```diff
@@ -517,16 +520,18 @@ def calc_alan_entries(
         Line opacity.
     """
 
-    phis = np.zeros(len(delta_nus))
-
-    for k in range(len(delta_nus)):
-        delta_nu = np.abs(delta_nus[k])
-        doppler_width = doppler_widths_at_depth_point[k]
-        gamma = gammas_at_depth_point[k]
+    delta_nus = np.abs(delta_nus)
+    # The above line can be removed if absolute value is not physical
 
-        phis[k] = voigt_profile(delta_nu, doppler_width, gamma)
+    std_dev = doppler_widths_at_depth_point * INV_SQRT_2
```

Comment:
@smithis7 can you please remind me where you found these formulas?

Reply:
@jvshields If these corrections are incorrect, I can remove them

Reply:
I believe they are incorrect. What we have currently produces reasonable voigt profiles.